tools/DevicesApiTester/Commands/Gpio/GpioButtonEvent.cs

```diff
@@ -111,6 +111,6 @@ void Console_CancelKeyPress(object sender, ConsoleCancelEventArgs e)
             }
         }
 
-        private bool OffValue => !OnValue;
+        private int OffValue => OnValue == 0 ? 1 : 0;
```

Comment:
nit: You can also do: 1 - OnValue

Reply:
That's a neat little trick 👍. Since the following is legal to make a PinValue.High...

```csharp
PinValue pinValue1 = -2;
PinValue pinValue1 = 2;
```
I added the following where used to force to 1 if not 0 at beginning of execution.
```csharp
if (OnValue != 0)
{
    OnValue = 1;
}
```